entrypoint.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 
-zip -r /tmp/training.zip /src
 mkdir -p /tmp/hpf/
 
-gunicorn --pythonpath /src -b 0.0.0.0:6006 --workers=1 -k sync -t 900 flask_endpoint:app
+gunicorn --pythonpath /opt/app-root/src/src -b 0.0.0.0:6006 --workers=1 -k sync -t 900 flask_endpoint:app --log-level $FLASK_LOGGING_LEVEL
```

Comment:
Not necessary to give the absolute path, please check https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/scripts/entrypoint.sh